homeassistant/components/verisure/climate.py

```diff
@@ -0,0 +1,218 @@
+"""Support for Verisure heatpump."""
+import logging
+from homeassistant.components.climate import ClimateDevice
+from homeassistant.components.climate.const import (
+    SUPPORT_TARGET_TEMPERATURE,
+    SUPPORT_FAN_MODE, SUPPORT_OPERATION_MODE,
+    SUPPORT_SWING_MODE, SUPPORT_ON_OFF)
+from homeassistant.const import TEMP_CELSIUS, ATTR_TEMPERATURE
+from homeassistant.components.verisure import HUB as hub
+from homeassistant.components.verisure import CONF_CLIMATE
+
+_LOGGER = logging.getLogger(__name__)
+HEAT_PUMPS = None
+VERISIRE_HASS_OP_MODE = {
+    'AUTO': 'auto',
```

Comment:
Import the base climate component operation states and use here for the home assistant side.